### ISSUE TITLE
[FEATURE] Apple PrivacyManifest

### ISSUE THREAD

**Issue by trevor-holliday-instacart** [verygoodsecurity/vgs-collect-ios#376]
**Is your feature request related to a problem? Please describe.**
Will VGS require an update to include a PrivacyManifest?

**Comment by dmytrokhl** [comment]
@trevor-holliday-instacart we will check this requirement. As for now I see this is optional for `VGSCollectSDK`: 

> Adoption is optional unless your - app is using a privacy-impacting SDK or - SDK is classified as a privacy-impacting SDK

List of privacy-impacting SDKs:  https://developer.apple.com/support/third-party-SDK-requirements/

**Comment by trevor-holliday-instacart** [comment]
@dmytrokhl that is a common list of SDKs, not an encompassing list. 

If VGS is not using any of the privacy-impacting apis it wont require a manifest.

**Comment by dmytrokhl** [comment]
We don't use privacy-impacting apis. But we will work on adding default PrivacyManifest.

**Comment by trevor-holliday-instacart** [comment]
Does VGS need to report that is collects financial information? 


Data type | Description | Value
-- | -- | --
Payment info | Such as form of payment, payment card number, or bank account number. If your app uses a payment service, the payment information is entered outside your app, and you as the developer never have access to the payment information, it is not collected and does not need to be disclosed. | NSPrivacyCollectedDataTypePaymentInfo

**Comment by icecoffin** [comment]
> We don't use privacy-impacting apis. But we will work on adding default PrivacyManifest.

The `VGSBlinkCardCollector` target is using the exact 2.7.0 version of `BlinkCard`, which doesn't have a privacy manifest. They added it in in 2.9.1. Would it be possible to update to the latest version (or just change from "exact" to "up to next major")? Runnin the [scanner](https://github.com/Wooder/ios_17_required_reason_api_scanner) shows some required reason API usage in BlinkCard.

**Comment by trevor-holliday-instacart** [comment]
@dmytrokhl is there an update on the status for the VGS privacy manifest?

**Comment by dmytrokhl** [comment]
@trevor-holliday-instacart VGS customers utilize our collect SDKs for a variety of use cases ranging from securing financial information to securing other types of sensitive data and we don’t want to create downstream problems for Customer end-users by inadvertently making our PrivacyManifest overly inclusive. 

We will add manifest file that speaks only to our common data collection practices that apply to all clients(e.g. analytics). 
So basically you will need to add manually to your App’s  manifest file additional data type depending on your use-case(Contact info, Financial info, etc.).

**Comment by dmytrokhl** [comment]
@trevor-holliday-instacart @icecoffin this is released in [v1.16.1](https://github.com/verygoodsecurity/vgs-collect-ios/releases/tag/1.16.1). 
You can read more about VGS Collect privacy manifest file [here](https://www.verygoodsecurity.com/docs/vgs-collect/ios-sdk/privacy-details).
